systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -253,4 +254,34 @@ public void testClusterWithCustomParameters() {
         assertThat(jsonString, valueOfCmEquals("KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "5"));
         assertThat(jsonString, valueOfCmEquals("KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", "5"));
     }
+    @Test
+    @Resources(value = "../examples/templates/cluster-controller", asAdmin = true)
+    @OpenShiftOnly
+    public void testDeployKafkaOnPersistentStorage() {
+        int expectedZKPods = 3;
+        int expectedKafkaPods = 3;
+        Oc oc = (Oc) this.kubeClient;
+        String clusterName = "openshift-my-cluster";
+        oc.newApp("strimzi-persistent", map("CLUSTER_NAME", clusterName));
+        oc.waitForStatefulSet(zookeeperStatefulSetName(clusterName), expectedZKPods);
+        oc.waitForStatefulSet(kafkaStatefulSetName(clusterName), expectedKafkaPods);
+
+        List<String> volumes = oc.list("pvc");
```

Comment:
Should we filter the PVC list based on its name to make sure that we count only the relevant PVCs? (the PVC created by the statefulset should contain the statefulset name).